src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/EverythingOperationContinuationToken.cs

```diff
@@ -31,14 +61,22 @@ public static EverythingOperationContinuationToken FromString(string json)
                 return null;
             }
 
-            try
-            {
-                return JsonSerializer.Deserialize<EverythingOperationContinuationToken>(json);
-            }
-            catch (JsonException)
+            return JsonConvert.DeserializeObject<EverythingOperationContinuationToken>(json);
+        }
+
+        public void AddSeeAlsoLink(string link)
+        {
+            // Safe to linear search on addition as it isn't common for a patient to have many "seealso" links
+            if (!SeeAlsoLinks.Contains(link))
             {
-                return null;
+                SeeAlsoLinks.Add(link);
```

Comment:
If there are many "seealso" links for one patient, there are chances that the length of the serialized continuation token exceeds the limit of the url. We may need to handle that case.

Reply:
We could consider adding a limit of the maximum number of the "seealso" links. By doing this we can ensure that the continuation token won't be too long to exceed the limit of the url.

Reply:
Thanks for bringing this up - I agree it would be good to limit the number of `seealso` links. For now, I plan to return an error if the limit is reached, keeping in mind that we are assuming that this isn't a very common scenario (@CaitlinV39 ran this assumption by the Smoke Jumpers).

Also note: @Ivanidzo4ka recommended adding logic to retrieve the parent patient and to extract the remaining links once the limit is reached. I created a story for that: [AB#85289](https://microsofthealth.visualstudio.com/Health/_workitems/edit/85289/)

Reply:
I wasn't convinced we had much storage space left in the token, so I ended up fetching the parent patient for each `seealso` link in a8f5b52.

A bit more info: the internal continuation token can be up to 2kB and resource ids can be up to 64 characters (64B). From my research, a URL has a max of 2083 characters, which doesn't give us much extra space (please let me know if I made any false assumptions here).